### ISSUE TITLE
Minified PHP files issue

### ISSUE THREAD

**Issue by chris-brabender** [magento/magento2#3893]
When magento 2 minifies PHP files into the view_preprocessed folder it seems to have a bit of a bug when it comes to comments.

If a PHP file contains a commented line starting with # or // it comments out all subsequent lines. 

EG:

```
# commented out line
$newVar = 'blah';
```

Gets minified into this:

```
# commented out line $newVar = 'blah';
```

Which effectively comments out everything after the commented line.

Is there a requirement to minify PHP blocks in these files?

**Comment by chris-brabender** [comment]
I have found another instance of this that is slightly different.

A template for a third party module we've been working with has something along the lines of this in javascript in a template file:

```
s.src = '//<?php echo $block->getSomevar() ?>.example.com/jsfile.js';
```

When magento minifies this it becomes

```
s.src = ' ?>.example.com/jsfile.js';
```

**Comment by PieterCappelle** [comment]
This issue is related with my issue #4490.

**Comment by anton-huz** [comment]
Hi @chris-brabender ,
You can find that issue was fixed for Magento 2.1 under #4365.

Please retest you own. I'm going to close issue Fill free to comment here some issues still exist.
